mk/cmd.mk

```diff
@@ -18,5 +18,11 @@ $(CMD): $(SRCS) $(CMD_DEPS) ; go build -o $(CMD)
 
 CLEANFILES += $(CMD)
 
+.PHONY: realall
+realall: $(CMD)
+
+.PHONY: cmd-hook-pre
+cmd-hook-pre:
```

Comment:
Where is the actual command for cmd-hook-pre?

Reply:
The _`<target name>`_+`-hook-pre` target is empty and can be (if needed) overridden by the caller to specify custom actions that need to happen before _`<target name>`_ is executed.